cypress-io/cypress
# feat: add support for next 13

### User facing changelog
Support Next.js v13

### Additional details
Bring our current support for Next.js 11 + 12 to v13. No support for `turbopack` or React SC (yet).

### Steps to test
- Run `npx create-next-app@latest my-app`
- cd into app and run `npm i -D cypress`
- Check out this branch, run `yarn && yarn cypress:open`, choose `my-app`
- Walk through setup
- Add a test, I added `pages/index.cy.tsx` and run it

```ts
import Home from './index'

it('should render', () => {
  cy.mount(<Home />)
})
```

### How has the user experience changed?

https://user-images.githubusercontent.com/25158820/200433321-c23a3164-93d5-4c18-83a2-223bb74022cb.mov

### PR Tasks
